documentation/docs/tutorials/validate_university_degree.mdx

```diff
@@ -102,25 +102,25 @@ After this step, you will find Alice's weakhold file in `./weakhold/Alice.json`.
 
 ### 2. **Issuer**: Create a DID
 
-Once you have created the Alice's DID(Decentralized Identifier), you should do the same for the University of Oslo.
+Once you have created Alice's DID (Decentralized Identifier), you should do the same for the University of Oslo.
 
-After this step, you will find the University of Oslo's weakhold file in `./weakhold/UniversityofOslo.json`.
+After finishing Alice's DID, you will find the University of Oslo's Weakhold file in `./weakhold/UniversityofOslo.json`.
 
-- [createDid.js](https://github.com/adrian-grassl/iota-identity-tutorial/blob/master/createDid.js)
+- [createDid.js](https://github.com/adrian-grassl/iota-identity-tutorial/blob/master/createDid.js).
 
 ```javascript
  createDid('University of Oslo');
 ```
 
 ### 3. **Issuer**: Add a Verification Method
 
-Since the university will need to verify Alice's degree, you should add a "degreeVerifications" verification method to the University's DID document.
+Since the University will need to verify Alice's degree, you should add a "degreeVerifications" verification method to the University's DID document.
 
 The University will have to sign more than just Alice's degree, so you should generate this verification method with a set of Merkle keys. These signatures can all be proved by a single public key, while retaining the ability to revoke them separately.
 
-Note that the newly added verification method is of the *type* `MerkleKeyCollection`.
+You should know that the newly added verification method is of the *type* `MerkleKeyCollection`.
```

Comment:
Why should I? Why "note something" doesn't work?

Reply:
Makes it more personable. @lucas-tortora minor consensus request.

Reply:
I guess we could split the difference and go for "You should note".  If taken out of context, "You should know" could be interpreted as "you should already know this", but given the context both work the same for me.

Reply:
Compromise achieved 🤝